draft-kwbdgrr-tsvwg-net-collab-rqmts.md

```diff
@@ -619,65 +606,53 @@ Use cases:
+The application server can decide on the metadata values that provide the best handling for the packets of the flow and may not necessarily reflect the exact priority values that allow an on-path observer to perform traffic analysis.
+This metadata is advisory in nature and network traffic policy ({{policy}}) that restricts its use would not result in additional issues.
+Other constraints including scale ({{scalability}}) and continuity ({{continuity}}) are required for {{relative-priority}} and {{delay}}.
 
-### Identification of Traffic Type without Disclosure of the Application {#TrafficType}
+Realizing the additional bandwidth potential with these metadata may require a higher sending rate for the transport flow.
+This requires work that is not specified in this document. Similarly, the assumption is that network shapers and schedulers can use the metadata in {{relative-priority}} and {{delay}} but further details are out of scope.
 
-Different nature/types of traffic can be part of the same 5-tuple flow. This could be reliable/loss-tolerant {{?RFC9221}}, bulk/interactive traffic. The type of traffic can be used to prioritize/buffer packets as needed and deprioritize/discard appropriate packets during reactive events, thereby optimizing performance. The application may provide information to identify the type of traffic in per-packet metadata.
+Previous work in {{TR.23.700-70-3GPP}} has identified the general problem in this section.
+However, the solution in {{TS.23.501-3GPP}} is specific to a 5G network.
+The metadata sent from a (dedicated 5G) application server identifies PDU set information and end-of-burst signals which are not understood by non-3GPP systems such as Wi-Fi or DOCSIS.
+Further, 3GPP functions and policy configurations are required since this is a 5G specific solution.
+The metadata disclosed in the 5G solution also identifies frame boundaries and does not fully conform to the constraints for privacy or minimality of data identified in {{operational}}.
 
-Requirements: REQ-PACKET-RELIABILITY, REQ-PACKET-NATURE as defined in {{mixed-traffic}}.
+### Packet Priority {#relative-priority}
 
-### Relative Priority {#relative-priority}
+Per-packet priority information provides the priority level of one packet relative to other packets within a transport flow (UDP 4-tuple).
+The application server can decide on the priority or importance values that provide the best handling for the packets of the transport flow and may not necessarily reflect the exact priority values that allow an on-path observer to perform traffic analysis.
+For example, when the application determines that congestion is very low, it may decide to mark all (or most) packets as high priority and when conditions change the server may mark more packets at a lower priority so as to maximize application performance and allowing a shaper to drop lower priority packets.
```

Comment:
Erm, I don't agree this would be a good example.  This would require more than one round trip for server to notice the changed congestion level after the client complains about loss or excessive ECN-marked packets received, requiring the client to then signal the server (via HTTP, updated SDP, or similar).   Why not have the server send packets marked differently the whole time?  Doing so would allow the network to drop less-important packets or forward important packets without suffering the greater-than-one-round-trip (*) delay.

(*) greater than one round trip because a reasonable client will not react to a single packet loss or single ECN-marked packet, but rather to several.

Reply:
Updated the revision to remove the text as suggested.